src/Sulu/Bundle/PageBundle/EventListener/PageTrashSubscriber.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\EventListener;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sulu\Bundle\PageBundle\Document\BasePageDocument;
+use Sulu\Bundle\TrashBundle\Application\TrashManager\TrashManagerInterface;
+use Sulu\Component\DocumentManager\Event\FlushEvent;
+use Sulu\Component\DocumentManager\Event\RemoveEvent;
+use Sulu\Component\DocumentManager\Events;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class PageTrashSubscriber implements EventSubscriberInterface
```

Comment:
```suggestion
/**
 * @internal
 * @final
 */
class PageTrashSubscriber implements EventSubscriberInterface
```

Would mark this as internal and final.

Reply:
👍